Author: Daniel Baumann <mail@daniel-baumann.ch>
Description: Adding aliases to the systemd service so they mask the LSB init scripts.

diff -Naurp plymouth.orig/systemd-units/plymouth-quit.service.in plymouth/systemd-units/plymouth-quit.service.in
--- plymouth.orig/systemd-units/plymouth-quit.service.in
+++ plymouth/systemd-units/plymouth-quit.service.in
@@ -7,4 +7,5 @@ ExecStart=-@PLYMOUTH_CLIENT_DIR@/plymout
 Type=oneshot
 TimeoutSec=20
 [Install]
+Alias=plymouth.service
 WantedBy=multi-user.target
diff -Naurp plymouth.orig/systemd-units/plymouth-read-write.service.in plymouth/systemd-units/plymouth-read-write.service.in
--- plymouth.orig/systemd-units/plymouth-read-write.service.in
+++ plymouth/systemd-units/plymouth-read-write.service.in
@@ -9,4 +9,5 @@ ConditionPathExists=!/etc/initrd-release
 ExecStart=-@PLYMOUTH_CLIENT_DIR@/plymouth update-root-fs --read-write
 Type=oneshot
 [Install]
+Alias=plymouth-log.service
 WantedBy=sysinit.target
